madrepo.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+set -e
+echo 'deb Https://sourceforge.net/projects/madlinux/files/repo core main'|sudo tee /etc/apt/sources.list.d/madlinux.list
+wget -qO- Https://sourceforge.net/projects/madlinux/files/repo/madlinux.key|gpg --dearmor|sudo tee /etc/apt/trusted.gpg.d/madlinux.gpg>/dev/null
+sudo add-apt-repository -y ppa:apt-fast/stable
+sudo apt install -y apt-fast
```

Comment:
Do you really think adding apt fast it's necessary? Because this would be one more dependency to install heroic.

Reply:
SourceForge is very slow in Brazil, that's why i include it.

Reply:
yes but since this is not a rule of thumb everywhere, maybe we should add information that using apt-fast might be better if the downloads are too slow.
And we can check on the script if there is apt-fast or not, if yes, we use it, if not, use normal apt. What do you think?

Reply:
Well, it can be, i'll made the changes.